### ISSUE TITLE
WSL integration doesn't create a symlink for docker-buildx

### ISSUE THREAD

**Issue by jandubois** [rancher-sandbox/rancher-desktop#3912]
It only links `docker-compose`:

```console
jan@DESKTOP-4992680:~$ ls ~/.docker/cli-plugins/
docker-compose
```

**Comment by adamkpickering** [comment]
I can repro this, assuming that the `~/.docker/cli-plugins/` directory we're talking about is one in a WSL distro we're integrating with.

**Comment by jandubois** [comment]
> assuming that the `~/.docker/cli-plugins/` directory we're talking about is one in a WSL distro we're integrating with.

Correct. I already looked at the code, and it seems to only deal with `docker-compose`.
